src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -102,25 +105,90 @@ const TeamMembers = createReactClass({
         });
       },
     });
+
+    this.api.request(`/organizations/${params.orgId}/members/`, {
+      success: data => {
+        this.setState({
+          orgMemberList: data,
+        });
+      },
+      error: () => {
+        IndicatorStore.add(t('Unable to load organization members.'), 'error', {
+          duration: 2000,
```

Comment:
this duration seems short, maybe make it 10,000 or leave it off and use the default?

Reply:
i think that is the default for the shortcut methods https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/stores/indicatorStore.jsx#L16